robotiq_c_model_visualization/package.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<package>
+  <name>robotiq_c_model_visualization</name>
+  <version>1.0.0</version>
```

Comment:
Does this version match the versions of all other robotiq packages?  It may not matter since the `robotiq` repo has never been release.  It will make this easier though.